src/services/conversations/conversations.hooks.js

```diff
@@ -82,9 +83,12 @@ const restrictAndSetOwner = () => context => {
           context.data.performedByRole = 'Campaign Reviewer';
           break;
         default:
-          throw new errors.Forbidden(
-            'Only people involved with the milestone can create conversation',
-          );
+          // TODO It's when creating donated conversation the ownerAddress maybe it's not
+          // involved to milestone, so we should not throw exception in this case
+          context.data.performedByRole = ' ';
```

Comment:
We threw error ton not allow anyone create conversation message. We can change it to only allow donate conversations be created by internal requests (see how it's handled src/hooks/onlyInternal.js)

Reply:
@aminlatifi  
Sure, I'll do it

Reply:
@aminlatifi 
Now I checked if request is not internal, I throw exception